internal/harness/golang/matchfunction_service.go

```diff
@@ -93,22 +91,11 @@ func (s *matchFunctionService) Run(req *pb.RunRequest, mfServer pb.MatchFunction
 		PoolNameToTickets: poolNameToTickets,
 	}
 	// Run the customize match function!
-	matchProposals := s.function(mfView)
-
-	// Send the proposals back in stream
-	for _, match := range matchProposals {
-		select {
-		case <-ctx.Done():
-			return nil
-		default:
-			err := mfServer.Send(&pb.RunResponse{Proposal: match})
-			if err != nil {
-				matchfunctionLogger.WithError(err).Error("Failed to send Run response to grpc server.")
-				return status.Error(codes.Aborted, err.Error())
-			}
-		}
-	}
-	return nil
+	proposals := s.function(mfView)
```

Comment:
Was deleting this code intentional?

Reply:
Yes  - previous one fetched and streamed each match back - now we return the set of matches.